psycopg/psycopg/pq/_enums.py

```diff
@@ -31,6 +31,7 @@ class ConnStatus(IntEnum):
     GSS_STARTUP = auto()
     CHECK_TARGET = auto()
     CHECK_STANDBY = auto()
+    ALLOCATED = auto()  # Only for cancel connections.
```

Comment:
Something broke in the docs generation (in the horrible `docs/lib/pg3_docs.py`) a while ago: the "docstring" style constants were published (e.g. [here](https://www.psycopg.org/psycopg3/docs/api/pq.html#psycopg.pq.ConnStatus.OK)).

However, the libpq declares that `ALLOCATED` is visible, therefore I would document it consistently:

```suggestion
    ALLOCATED = auto()
    """Connection to the server hasn't been initiated yet."""
```

The blurb:

https://github.com/psycopg/psycopg/blob/cae87c4cf6223cc278f36759ea8709cc628ac7f6/docs/api/pq.rst?plain=1#L148-L150

might also require adjustment, for instance:

```
    There are other values in this enum, but only `ALLOCATED`, `OK`, `BAD` 
    are seen after a connection has been established. Other statuses might
    only be seen during the connection phase and are considered internal.
    `!ALLOCATED` is only expected to be returned by `PGcancelConn.status`.
```

Reply:
Done as well.